src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -214,6 +214,7 @@ message RedisShardBackplaneConfig {
   int32 action_cache_expire = 4;
   string action_blacklist_prefix = 28;
   int32 action_blacklist_expire = 29;
+  string invocation_blacklist_prefix = 30;
```

Comment:
do we want to use `repeated`?

Reply:
Negative, why would you want to specify multiple invocation blacklist prefixes?

Reply:
oops. misread this as actual blacklisted invocations